topcoffea/json/rate_systs.json

```diff
@@ -35,7 +34,7 @@
         "tllq":     {"pdf_scale": "qq", "qcd_scale": "V"   },
         "Diboson":  {"pdf_scale": "qq", "qcd_scale": "VV"  },
         "Triboson": {"pdf_scale": "qq", "qcd_scale": "VVV" },
-        "convs":    {"pdf_scale": "gg", "qcd_scale": "ttG" }
+        "convs":    {"pdf_scale": "gg", }
```

Comment:
Why was the QCD scale removed?

Reply:
For ttgamma, we don't have an NLO xsec for this sample, so we're using the LO one from the MG gridpack. Since it's an LO xsec, we would want to calculate the uncertainty via the renorm/fact and ISR/FSR systematics (if I'm understanding correctly the explanation from @klannon). Usually (when we have an NLO xsec) for these uncertainties do this (e.g. for the up variation):
`xsec_NLO*(N_pass_up/N_gen_up)`
As this allows us to account for the shape differences, while leaving the xsec alone (since the uncertainty on that is taken care of with the rate uncertainty in the datacard). However, in the case where we have an LO xsec, we can do this:
`xsec_NLO*(N_pass_up/N_gen)`
As this will include both the shape and rate uncertainties. Since we're estimating the uncertainty on the LO xsec in this way, we would not want to also apply the NLO xsec uncertainty on top of that. 

I've attempted to also explain this in the processor [here](https://github.com/TopEFT/topcoffea/blob/27d1c0e25eef80a59f042e57757294d962c6a76f/analysis/topEFT/topeft.py#L126-L154) where I've modified the up/down sum of weights to be the nominal sum of weights (in the case of the ttgamma LO samples).

Reply:
Right, I think so.